ios/PolyPodApp/PolyPod/SettingsView.swift

```diff
@@ -107,10 +108,16 @@ private struct MainSection: View {
                     label: "settings_licenses_title",
                     action: { activeSection = .licenses }
                 )
+                SettingsButton(label: "Export Logs",
```

Comment:
Most probably needs localisation.

Reply:
If this is only for testers and TestFlight users, probably not...